app/models/category.rb

```diff
@@ -1,3 +1,5 @@
 class Category < ApplicationRecord
   has_many :tasks
+
+  validates :name, presence: true
 end
```

Comment:
@jmejia that's a good point. I think we might hold off based on Rachel's slides, but we will bring that up when we talk to her today. Thanks!